stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/util/ImageUtils.kt

```diff
@@ -161,7 +169,15 @@ internal fun StreamAsyncImage(
                 imageLoader = LocalStreamImageLoader.current,
                 contentScale = contentScale,
             )
-            content(asyncImagePainter.state)
+            val state: AsyncImagePainter.State by asyncImagePainter.state.collectAsState()
```

Comment:
nit: declaring the type just seems to create noise.
`val state by asyncImagePainter.state.collectAsState()` should be enough.